liquibase-cli/pom.xml

```diff
@@ -23,6 +23,8 @@
             <groupId>info.picocli</groupId>
             <artifactId>picocli</artifactId>
             <version>4.7.1</version>
+            <scope>compile</scope>
+            <optional>true</optional>
```

Comment:
Is pico really an optional dependency? Seems pretty non-optional to me.

Reply:
The reason that picocli got marked as optional is because we don't want liquibase-core to have that as a dependency. It currently isn't there at all, because it is optional for liquibase-core.
Ideally it would come through as optional vs. not there at all and my hope was that marking it as optional in cli would bubble through to liquibase-core that way, but it just doesn't show up at all.
So it's in sort of a "I wish it worked this way" state, but also very easy for people to accidentally break by removing that "optional" flag in liquibase-cli (since that would make sense) and then suddenly we are releasing liqubiase-core with a hard dependency on picocli which we don't want.
So I pushed a change that makes it more explicit that liquibase-core doesn't include picocli as a dependency.

Reply:
I think the change d7307d98439c9ae17accd96d557e9ffbf530d5e3 makes your intention much clearer. Thanks!